Engine/source/scene/pathManager.cpp

```diff
@@ -299,15 +303,33 @@ void PathManager::getPathPosition(const U32 id,
 
    // Ok, query holds our path information...
    F64 ms = msPosition;
-   if (ms > mPaths[id]->totalTime)
-      ms = mPaths[id]->totalTime;
+
+   //Looping vs. non-looping splines
+   F32 msTotal;
+   if (mPaths[id]->looping)
+      msTotal = mPaths[id]->totalTime;
```

Comment:
I know that this is similar to how the code was written before, but I believe our official code-style is to always include braces?